biojava-structure/src/main/java/org/biojava/nbio/structure/contact/Pair.java

```diff
@@ -97,4 +97,14 @@ public boolean equals(Object obj) {
 	public String toString() {
 		return "<" + first.toString() + ", " + second.toString() + ">";
 	}
+
+	/**
+	 * Tells whether the interface corresponds to one mediated by crystallographic symmetry,
+	 * i.e. it is between symmetry-related molecules (with same chain identifier)
+	 * @return
+	 */
+	public boolean isSymRelated() {
```

Comment:
The Pair class is general and intended for any kind of pair (note the generic type `T`). This would break that generality.